simulator/risc_v/t/unit_test.cpp

```diff
@@ -358,6 +360,11 @@ TEST_RV64_RR_OP( 1, add_uw, 0x12344321b5a69788, 0xabababab82736455, 0x1234432133
 TEST_RV64_RR_OP( 2, add_uw, 0x1111111233333332, 0x11111111ffffffff, 0x1111111133333333) // 32 bits overflow
 TEST_RV64_RR_OP( 3, add_uw, 0x01010102222221ce, 0xffffffffffffffac, 0x0101010122222222) // 64 bits overflow
 
+TEST_RV64_IMM_OP ( 1, bseti, 0xA95, 0xA91, 0x2)
+TEST_RV64_IMM_OP ( 2, bseti, 0x159D26AF37BF48C0, 0x159D26AE37BF48C0, 0x20)
```

Comment:
This is a very unobvious test. What value I expect to see if it fails after my changes?
The tests should tell about the purpose of instruction w/o looking into spec. Check #1504, for instance.